projects/pluto_ng/system_bd.tcl

```diff
@@ -0,0 +1,603 @@
+
+ad_connect util_adc_1_pack/packed_fifo_wr axi_adrv9001_rx1_dma/fifo_wr
+
+# RX2 - CPACK - RX_DMA2
+ad_connect  axi_adrv9001/adc_2_rst       util_adc_2_pack/reset
+ad_connect  axi_adrv9001/adc_2_valid_i0  util_adc_2_pack/fifo_wr_en
+ad_connect  axi_adrv9001/adc_2_enable_i0 util_adc_2_pack/enable_0
+ad_connect  axi_adrv9001/adc_2_data_i0   util_adc_2_pack/fifo_wr_data_0
+ad_connect  axi_adrv9001/adc_2_enable_q0 util_adc_2_pack/enable_1
+ad_connect  axi_adrv9001/adc_2_data_q0   util_adc_2_pack/fifo_wr_data_1
+
+ad_connect  axi_adrv9001/adc_2_dovf       util_adc_2_pack/fifo_wr_overflow
+
+ad_connect util_adc_2_pack/packed_fifo_wr axi_adrv9001_rx2_dma/fifo_wr
+
+# TX_DMA1 - UPACK - TX1
+ad_connect  axi_adrv9001/dac_1_rst        util_dac_1_upack/reset
+ad_connect  axi_adrv9001/dac_1_valid_i0   util_dac_1_upack/fifo_rd_en
+ad_connect  axi_adrv9001/dac_1_enable_i0  util_dac_1_upack/enable_0
+ad_connect  axi_adrv9001/dac_1_data_i0    util_dac_1_upack/fifo_rd_data_0
+ad_connect  axi_adrv9001/dac_1_enable_q0  util_dac_1_upack/enable_1
+ad_connect  axi_adrv9001/dac_1_data_q0    util_dac_1_upack/fifo_rd_data_1
+ad_connect  axi_adrv9001/dac_1_enable_i1  util_dac_1_upack/enable_2
+ad_connect  axi_adrv9001/dac_1_data_i1    util_dac_1_upack/fifo_rd_data_2
+ad_connect  axi_adrv9001/dac_1_enable_q1  util_dac_1_upack/enable_3
+ad_connect  axi_adrv9001/dac_1_data_q1    util_dac_1_upack/fifo_rd_data_3
+
+ad_connect  axi_adrv9001_tx1_dma/m_axis   util_dac_1_upack/s_axis
+ad_connect  axi_adrv9001/dac_1_dunf       util_dac_1_upack/fifo_rd_underflow
+
+# TX_DMA2 - UPACK - TX2
+ad_connect  axi_adrv9001/dac_2_rst        util_dac_2_upack/reset
+ad_connect  axi_adrv9001/dac_2_valid_i0   util_dac_2_upack/fifo_rd_en
+ad_connect  axi_adrv9001/dac_2_enable_i0  util_dac_2_upack/enable_0
+ad_connect  axi_adrv9001/dac_2_data_i0    util_dac_2_upack/fifo_rd_data_0
+ad_connect  axi_adrv9001/dac_2_enable_q0  util_dac_2_upack/enable_1
+ad_connect  axi_adrv9001/dac_2_data_q0    util_dac_2_upack/fifo_rd_data_1
+
+ad_connect  axi_adrv9001_tx2_dma/m_axis   util_dac_2_upack/s_axis
+ad_connect  axi_adrv9001/dac_2_dunf       util_dac_2_upack/fifo_rd_underflow
+
+ad_connect  gpio_rx1_enable_in            axi_adrv9001/gpio_rx1_enable_in
+ad_connect  gpio_rx2_enable_in            axi_adrv9001/gpio_rx2_enable_in
+ad_connect  gpio_tx1_enable_in            axi_adrv9001/gpio_tx1_enable_in
+ad_connect  gpio_tx2_enable_in            axi_adrv9001/gpio_tx2_enable_in
+
+ad_connect  rx1_enable                    axi_adrv9001/rx1_enable
+ad_connect  rx2_enable                    axi_adrv9001/rx2_enable
+ad_connect  tx1_enable                    axi_adrv9001/tx1_enable
+ad_connect  tx2_enable                    axi_adrv9001/tx2_enable
+
+ad_connect  system_sync                   axi_adrv9001/adc_sync_in
+ad_connect  system_sync                   axi_adrv9001/dac_sync_in
+
+# sysytem monitor
+
+ create_bd_cell -type ip -vlnv xilinx.com:ip:system_management_wiz:1.3 system_management_wiz_0
```

Comment:
Can this be created using _ad_ip_instance_ ?

Reply:
Done. Also, I renamed system_management_wiz_0 to pl_sysmon.